### PR TITLE
Adds a help command within the interactive query prompt

### PR DESCRIPTION
**Pull request by dwelch-r7**
Adds the ability to run a `help` command from within the interactive SQL prompt

before:
```
SQL >> help
SQL *>
```

After:
```
SQL >> help
Interactive SQL prompt

You are in an interactive SQL shell where SQL queries can be executed.
To exit, type 'exit', 'quit', 'end' or 'stop'.
```

# Verification Steps
- [ ] boot msfconsole
- [ ] Get a SQL type session
- [ ] drop into an interactive prompt `query_interactive`
- [ ] run the `help` command

### CODE REVIEW

**Comment by sjanusz-r7** [comment]
@msjenkins-r7 retest this please

File: lib/rex/post/sql/ui/console/command_dispatcher/client.rb [hunk 1]
```diff
@@ -67,6 +67,14 @@ def cmd_query_interactive_help
                 print_line
               end
 
+              def query_interactive_help
+                print_line 'Interactive SQL prompt'
+                print_line
+                print_line 'You are in an interactive SQL shell where SQL queries can be executed.'
```
Comment (adfoster-r7) [inline lib/rex/post/sql/ui/console/command_dispatcher/client.rb:73]:
```suggestion
                print_line 'You are in an interactive SQL shell where SQL queries can be executed.'
                print_good('SQL commands ending with ; will be executed on the remote server.')

```

File: lib/rex/post/sql/ui/console/interactive_sql_client.rb [hunk 2]
```diff
@@ -100,10 +107,15 @@ def _multiline
...
-        finished = (multiline_input.split.count == 1 && stop_words.include?(multiline_input.split.last))
+        if multiline_input.split.count == 1
+          # In the case only a stop word was input, exit out of the REPL shell
+          finished = stop_words.include?(multiline_input.split.last)
+          # In the case when only a help word was input call the help command
+          help = help_words.include?(multiline_input.split.last)
+        end
 
         finished || multiline_input.split.last&.end_with?(';')
+        help || multiline_input.split.last&.end_with?(';')
```
Comment (adfoster-r7) [inline lib/rex/post/sql/ui/console/interactive_sql_client.rb:118]:
```suggestion
        finished || help || multiline_input.split.last&.end_with?(';')
```

Or something else maybe

**Comment by cgranleese-r7** [comment]
Tested against PostgreSQL 16.1 and works as expected 👍 

## Before
image


## After 
image

**Review by cgranleese-r7 — APPROVED** [review]

**Comment by cgranleese-r7** [comment]
## Release Notes
This PR adds the ability to run a help command from within the interactive SQL prompt.

**Pull request merged**